runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -114,16 +114,17 @@ public Layout getCurrentLayout() {
                             + "invalidate view", we.getCorrectEpoch());
                 } else if (re instanceof NetworkException) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the usage of the "instanceof" operator by a catch block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1193)